src/main/java/org/purescript/parser/PureParsecParser.kt

```diff
@@ -510,17 +510,34 @@ class PureParsecParser {
                         )
                     )
             ).`as`(TypeInstanceDeclaration)
-    private val importDeclarationType =
-        optional(indented(parens(commaSep(parseDeclarationRef))))
+    private val importedDataMembers = parens(
+        choice(
+            ddot,
+            commaSep(properName.`as`(PSElements.ImportedDataMember))
+        )
+    ).`as`(PSElements.ImportedDataMemberList)
+    private val importedItem =
+        choice(
+            token(TYPE).then(parens(operator.`as`(Identifier))).`as`(PSElements.ImportedType),
+            token(CLASS).then(properName).`as`(PSElements.ImportedClass),
+            token(KIND).then(properName).`as`(PSElements.ImportedKind),
+            parens(operator.`as`(Identifier)).`as`(PSElements.ImportedOperator),
```

Comment:
I parse operators as `Identifier` because I need them to more than just `PsiElement`. It's done elsewhere in the parser, but there might be a case for having a separate `PSOperator` element instead of reusing `PSIdentifierImpl`.